modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6140,6 +6140,8 @@ enable-email-domain-validation=Enable Email Domain Validation
 enable-email-domain-validation-help=Enable to allow entering a list of valid email domains when editing an account. Account users must have a matching email domain for self registration. Administrators can manually add account users with non-matching email domains.
 enable-email-subscription=Enable Email Subscription
 enable-entry-history=Enable Entry History
+enable-entry-translations=Enable Entry Translation
+enable-entry-translations-in-all-fields-go-to-field-settings-to-disable-translations-individually-this-configuration-cannot-be-changed-after-the-object-is-published=Enable entry translations in all fields. Go to field settings to disable translations individually. This configuration cannot be changed after the object is published.
```

Comment:
@danvb112 you can end the key in the first sentence, so it doesn't become so long.